server/data/nl/danielsjf.txt

```diff
@@ -3379,3 +3379,440 @@ De stelling van Pythagoras beschrijft de relatie tussen zijden in een driehoek.
+Ik heb een fiets gekregen voor mijn communie.
+Ken je haar achternaam?
+Zijn voornaam is Matthias.
+Is dat zijn bijnaam?
+Vroeger hadden we bijnamen voor iedereen in de klas.
+Toen ik jong was waren de computerlessen relatief simpel.
+Nu leren veel kinderen hoe ze moeten programmeren op school.
+Ik ben nooit veel naar de kerk geweest.
+Ga je soms naar de moskee?
+Bezoek je iedere week de synagoge?
+Hij citeert een stuk uit de Bijbel.
+Zij kent dat stuk uit de Koran vanbuiten.
+Hij leest uit de Thora.
+Het Christendom is een monotheïstische godsdienst.
+Is Frans een Romaanse taal?
+Het Romeinse Rijk strekte zich uit van Noord-Afrika tot West-Europa.
+Monaco is een stadstaat.
+Zijn vader had een mooie jacht. 
+Wat voor een lens heb je gekocht?
+Crème brûlé is mijn favoriete dessert.
+Mijn mama maakt altijd heerlijke tiramisu. 
+Ik lust wel een dame blanche.
+Met dit winterweer heb ik zin in een warme appeltaart.
+De bakker had verse kersentaart gemaakt.
+Kan je me twee sneden brood geven?
+In de bakkerij rook het naar vers gebakken brood. 
+De slager gaf het kind een stukje worst.
+Ze kochten hun melk altijd rechtstreeks van de boer. 
+Iedere ochtend stond de boerin op om de koeien te melken.
+Ze had dat pikant detail nooit verteld.
+De saus was te pikant.
+Er mogen nog wat kruiden in de puree. 
+Het aantal politieagenten was buiten proportie. 
+Banken moeten zich melden bij de nationale bank.
+Ik heb een nieuwe keuken gekocht op krediet.
+Hoeveel kredietkaart heb jij?
+Ik heb een mastercard, een visa en een maestro kaart.
+In het buitenland kan je gemakkelijk betalen met een kredietkaart.
+Let op voor de hoge kosten op een visakaart.
+Wat is de huidige wisselkoers tussen de Turkse lira en de euro?
+De Europese Centrale Bank, kortweg de ECB, bepaalt de monetaire koers in Europa.
+Een nationale bank wordt geacht onafhankelijk te zijn.
+De inflatie schommelt rond de twee procent.
+Hoeveel rente moet je betalen?
+De vissersboot had zich vastgevaren in het ondiepe water.
+Het kanaal was dichtgeslibd.
+Er bevindt zich teveel slib in de rivierbedding.
+Er is een ferry die je naar de overkant kan brengen.
+Er is een veerpont om fietsers te vervoeren.
+Zij sliep als een roosje.
+Samen zongen ze het volkslied.
+Ik heb een klavertje vier gevonden.
+Naar het schijnt brengt een klavertje vier geluk.
+Ze hield zich vast aan de steunpilaar.
+Ze kon het zware gewicht niet meer dragen.
+Ze struikelde maar kon zich nog net overeind houden. 
+Hij was moe van de lange tocht.
+De dief kon vluchten via het dak.
+Ze heeft de kleur veranderd met een natuurlijk pigment.
+Het biggetje huppelde door het lange gras.
+De verliefdheid geeft mij vlinders in mijn buik.
+Er werd een gevecht georganiseerd in de arena.
+Er zat een arend in de toren van de kerk.
+De uitdaging was aartsmoeilijk.
+Door de stress kon ik mij niet concentreren.
+De politie vergroot het zoekgebied stelselmatig.
+De figuur bestond uit een reeks concentrische cirkels. 
```

Comment:
Mijn gevoel zegt "het" maar volgens internet mag het zowel "het" als "de figuur" zijn..

Reply:
Ik zou sowieso ook het figuur gebruiken voor een lichaamsbouw. Voor een figuur als in een afbeelding mag je blijkbaar inderdaad beide gebruiken.